inc/namespace.php

```diff
@@ -17,20 +20,25 @@ function bootstrap() {
 	// Enqueue the javascript handler.
 	add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\enqueue_assets' );
 
-	// Shortcode. Replace with an actual way to display a banner.
+	// Shortcode. Not recommended but here in case it's needed.
 	add_shortcode( 'cookie-consent-banner', __NAMESPACE__ . '\\render_consent_banner' );
 }
 
+/**
+ * Enqueue css and js.
+ */
 function enqueue_assets() {
-	$js = plugin_dir_url( __DIR__ ) . 'dist/js/main.js';
+	$js  = plugin_dir_url( __DIR__ ) . 'dist/js/main.js';
+	$ver = '1.0.0';
 
 	// If working locally, load the unminified version of the js file.
 	if ( Altis\get_environment_type() === 'local' ) {
-		$js = plugin_dir_url( __DIR__ ) . 'assets/js/main.js';
+		$js   = plugin_dir_url( __DIR__ ) . 'assets/js/main.js';
+		$ver .= '-' . time();
```

Comment:
Could just set the version to `null` here to avoid generating the query string entirely. Perhaps on local you could use `filemtime()` instead. Usually when working on JS I have the browser dev tools open which allows to switch the cache off while it's open. Avoids the need for anything like this.

Reply:
Yeah, I was thinking about cache-breaking stuff here. `filemtime` is a much better solution than `time` 👍